apps/web/app/components/navbar/Navbar.tsx

```
@@ -0,0 +1,107 @@
+
+export default function Navbar() {
+    const pathname = usePathname();
+    const [scrolled, setScrolled] = useState(false);
+    const [overlayOpen, setOverlay] = useState(false);
+    const [activeLink, setActiveLink] = useState<string | null>(null);
+    const rafRef = useRef<number>(0);
+
+    // Hide on auth pages
+    if (pathname?.startsWith("/sign-in")) return null;
+
+    useEffect(() => {
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)